coderd/workspaces_test.go

```diff
@@ -1510,8 +1510,11 @@ func TestWorkspaceFilterManual(t *testing.T) {
 
 	t.Run("LastUsed", func(t *testing.T) {
 		t.Parallel()
+		db, ps := dbtestutil.NewDB(t, dbtestutil.WithTimezone("UTC"))
```

Comment:
review: LastUsed is apparently `TIMESTAMP WITHOUT TIME ZONE`. Not sure why this is.

Reply:
There's no comment on the DB field explaining the reasoning, so it's definitely a bug.

The migration that added it made a mistake of using `timestamp` vs `timestamptz`.

```
000043_workspace_last_used.up.sql
2:    ADD COLUMN last_used_at timestamp NOT NULL DEFAULT '0001-01-01 00:00:00+00:00';
```

We should probably lint this somehow and prevent addition of `timestamp` columns unless `//nolint`ed.

Reply:
Same bug in `users` table with `last_seen_at`.

Reply:
Filed https://github.com/coder/coder/issues/9682 to follow up.